app/models/tag.rb

```diff
@@ -73,7 +75,7 @@ def sorted_children
     if child_ordering == "alphabetical"
       children.sort_by { |child| child.title.downcase }
     else
-      children.sort_by(&:index)
+      children.map(&:index).any?(&:nil?) ? children : children.sort_by(&:index)
```

Comment:
Could this be refactored into the main `if` statement?

I can see what this line is doing, but am wondering whether it'd be more readable as a three-part `if` statement rather than nesting it.

e.g.

```ruby
if child_ordering == "alphabetical"
  children.sort_by { |child| child.title.downcase }
elsif children.map(&:index).all?(&:present?)
  children.sort_by(&:index)
else
  children
end
```